### PR TITLE
Create issue templates for different use cases

### LINKED ISSUE #305: Create issue templates for different use cases.
E.g. doc issue
code issue etc.

Always require the OS, version, py version, command/code tried as basic information.

### PR DESCRIPTION
**Pull request by Happy-reindeer007**
Creates bug report and documentation issue templates

Closes #305

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a report when something isn't working
+title: ''
+labels: ''
```
Comment (jpfeuffer) [inline .github/ISSUE_TEMPLATE/bug-report.md:5]:
```suggestion
labels: 'bug'
```

File: .github/ISSUE_TEMPLATE/documentation-issue.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: Documentation issue
+about: Create a report if there is a problem with the documentation
+title: ''
+labels: ''
```
Comment (jpfeuffer) [inline .github/ISSUE_TEMPLATE/documentation-issue.md:5]:
```suggestion
labels: 'documentation'
```

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+**What should be happening**
+<!-- Please tell us briefly what was supposed to happen -->
+
+**Screenshots**
+<!-- Please add a screenshot showing the bug -->
+
+**System information:**
+ - OS:
+ - Version:
```
Comment (jpfeuffer) [inline .github/ISSUE_TEMPLATE/bug-report.md:28]:
```suggestion
 - OS Version:
```

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+ - Python version:
```
Comment (jpfeuffer) [inline .github/ISSUE_TEMPLATE/bug-report.md:29]:
```suggestion
 - Python version:
 - pyopenms version:
 - How did you install pyopenms? Please cross a box with an X.
     - [ ] conda::bioconda: Specify your conda/mamba command and/or your environment with `conda list`.
     - [ ] conda::openms: Specify your conda/mamba command and/or your environment with `conda list`.
     - [ ] pip
     - [ ] nightly wheel
     - [ ] built from source
```

**Review by jpfeuffer — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a report when something isn't working
```
Comment (jpfeuffer) [inline .github/ISSUE_TEMPLATE/bug-report.md:3]:
```suggestion
about: Create a report when something isn't working with your python code using pyopenms
```

File: .github/ISSUE_TEMPLATE/documentation-issue.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+---
+name: Documentation issue
+about: Create a report if there is a problem with the documentation
```
Comment (jpfeuffer) [inline .github/ISSUE_TEMPLATE/documentation-issue.md:3]:
```suggestion
about: Create a report if there is a problem with the documentation. E.g., unclear, typography, not reproducible or incomplete
```

**Review by jpfeuffer — CHANGES_REQUESTED** [review]

**Comment by jpfeuffer** [comment]
Thanks!!

**Comment by Happy-reindeer007** [comment]
You’re welcome, I’ve just applied the changes that you suggested.

BTW, I’ve just noticed that this repo hasn’t got the hacktoberfest label, even though the issue had it, so it’s not counting towards the event. Would you mind adding it? Thanks!

**Comment by jpfeuffer** [comment]
I added the label.
Btw did you know that you can add suggestions to a batch and commit them altogether?
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request

**Comment by Happy-reindeer007** [comment]
> Btw did you know that you can add suggestions to a batch and commit them altogether?
> https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request

Yeah, I tried to but for some reason it wasn’t letting me…

**Comment by Happy-reindeer007** [comment]
Hi there! Did you have time to take a look at the changes?

**Pull request merged**